src/pages/Background/index.js

```diff
@@ -1,6 +1,81 @@
+import 'regenerator-runtime/runtime.js';
+import axios from 'axios';
+import fetchAdapter from '@vespaiach/axios-fetch-adapter';
+import { createQAUrl } from '../Content/utiles/utiles.js';
 console.log('This is the background page.');
-console.log('Put the background scripts here.');
 
-chrome.action.onClicked.addListener((tab) => {
-    chrome.scripting.executeScript(tab.id, { file: 'contentScript.bundle.js'});
-  });
\ No newline at end of file
+// chrome.actiont.onClicked.addListener((tab) => {
+//   chrome.scripting.executeScript(tab.id, { file: 'contentScript.bundle.js' });
+// });
+
+let currentUrl, currentTabId;
+chrome.tabs.onUpdated.addListener(function (tabId, { url }, tab) {
```

Comment:
Do we really need to run this code in the background for every tab change? Can't we inject only when popup is triggered, and only for sites which have the isqa param?